Thanks for this PR.
I have a bunch of new warnings that I do not get on master, do you have them too ?
Any idea how to get rid of these ?

```
warning: In file included from /nix/store/b3zsk4ihlpiimv3vff86bb5bxghgdzb9-gcc-9.2.0/include/c++/9.2.0/x86_64-unknown-linux-gnu/bits/os_defines.h:39,
warning:                  from /nix/store/b3zsk4ihlpiimv3vff86bb5bxghgdzb9-gcc-9.2.0/include/c++/9.2.0/x86_64-unknown-linux-gnu/bits/c++config.h:524,
warning:                  from /nix/store/b3zsk4ihlpiimv3vff86bb5bxghgdzb9-gcc-9.2.0/include/c++/9.2.0/bits/stl_algobase.h:59,
warning:                  from /nix/store/b3zsk4ihlpiimv3vff86bb5bxghgdzb9-gcc-9.2.0/include/c++/9.2.0/vector:60,
warning:                  from clipper/clipper.hpp:52,
warning:                  from clipper/clipper.cpp:41:
warning: /nix/store/fwpn2f7a4iqszyydw7ag61zlnp6xk5d3-glibc-2.30-dev/include/features.h:382:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning:   382 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning:       |    ^~~~~~~
warning: clipper/clipper.cpp: In function ‘ClipperLib::Int128 ClipperLib::Int128Mul(ClipperLib::long64, ClipperLib::long64)’:
warning: clipper/clipper.cpp:376:22: warning: implicitly-declared ‘constexpr ClipperLib::Int128& ClipperLib::Int128::operator=(const ClipperLib::Int128&)’ is deprecated [-Wdeprecated-copy]
warning:   376 |   if (negate) tmp = -tmp;
warning:       |                      ^~~
warning: clipper/clipper.cpp:264:5: note: because ‘ClipperLib::Int128’ has user-provided ‘ClipperLib::Int128::Int128(const ClipperLib::Int128&)’
warning:   264 |     Int128(const Int128 &val): lo(val.lo), hi(val.hi){}
warning:       |     ^~~~~~
warning: clipper/clipper.cpp: In function ‘void ClipperLib::InitEdge(ClipperLib::TEdge*, ClipperLib::TEdge*, ClipperLib::TEdge*, const ClipperLib::IntPoint&)’:
warning: clipper/clipper.cpp:721:34: warning: ‘void* memset(void*, int, size_t)’ clearing an object of non-trivial type ‘struct ClipperLib::TEdge’; use assignment or value-initialization instead [-Wclass-memaccess]
warning:   721 |   std::memset(e, 0, sizeof(TEdge));
warning:       |                                  ^
warning: clipper/clipper.cpp:66:8: note: ‘struct ClipperLib::TEdge’ declared here
warning:    66 | struct TEdge {
warning:       |        ^~~~~
warning: In file included from /nix/store/b3zsk4ihlpiimv3vff86bb5bxghgdzb9-gcc-9.2.0/include/c++/9.2.0/x86_64-unknown-linux-gnu/bits/os_defines.h:39,
warning:                  from /nix/store/b3zsk4ihlpiimv3vff86bb5bxghgdzb9-gcc-9.2.0/include/c++/9.2.0/x86_64-unknown-linux-gnu/bits/c++config.h:524,
warning:                  from /nix/store/b3zsk4ihlpiimv3vff86bb5bxghgdzb9-gcc-9.2.0/include/c++/9.2.0/cstdlib:41,
warning:                  from /nix/store/b3zsk4ihlpiimv3vff86bb5bxghgdzb9-gcc-9.2.0/include/c++/9.2.0/stdlib.h:36,
warning:                  from clipper/wrapper.h:8,
warning:                  from clipper/wrapper.cpp:1:
warning: /nix/store/fwpn2f7a4iqszyydw7ag61zlnp6xk5d3-glibc-2.30-dev/include/features.h:382:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning:   382 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning:       |    ^~~~~~~
    Finished test [unoptimized + debuginfo] target(s) in 3.59s
```

--- a/clipper/clipper.cpp
+++ b/clipper/clipper.cpp
@@ -264,6 +264,8 @@ class Int128
     Int128(const Int128 &val): lo(val.lo), hi(val.hi){}
 
     Int128(const long64& _hi, const ulong64& _lo): lo(_lo), hi(_hi){}
+
+    Int128& operator = (const Int128&) = default;
     
     Int128& operator = (const long64 &val)
     {
@@ -718,7 +720,7 @@ void DisposeOutPts(OutPt*& pp)
 
 inline void InitEdge(TEdge* e, TEdge* eNext, TEdge* ePrev, const IntPoint& Pt)
 {
-  std::memset(e, 0, sizeof(TEdge));
+  *e = TEdge();
   e->Next = eNext;
   e->Prev = ePrev;
   e->Curr = Pt;
@@ -4329,10 +4331,10 @@ double DistanceFromLineSqrd(
   const IntPoint& pt, const IntPoint& ln1, const IntPoint& ln2)
 {
   //The equation of a line in general form (Ax + By + C = 0)
-  //given 2 points (x¹,y¹) & (x²,y²) is ...
-  //(y¹ - y²)x + (x² - x¹)y + (y² - y¹)x¹ - (x² - x¹)y¹ = 0
-  //A = (y¹ - y²); B = (x² - x¹); C = (y² - y¹)x¹ - (x² - x¹)y¹
-  //perpendicular distance of point (x³,y³) = (Ax³ + By³ + C)/Sqrt(A² + B²)
+  //given 2 points (xï¿½,yï¿½) & (xï¿½,yï¿½) is ...
+  //(yï¿½ - yï¿½)x + (xï¿½ - xï¿½)y + (yï¿½ - yï¿½)xï¿½ - (xï¿½ - xï¿½)yï¿½ = 0
+  //A = (yï¿½ - yï¿½); B = (xï¿½ - xï¿½); C = (yï¿½ - yï¿½)xï¿½ - (xï¿½ - xï¿½)yï¿½
+  //perpendicular distance of point (xï¿½,yï¿½) = (Axï¿½ + Byï¿½ + C)/Sqrt(Aï¿½ + Bï¿½)
   //see http://en.wikipedia.org/wiki/Perpendicular_distance
   double A = double(ln1.Y - ln2.Y);
   double B = double(ln2.X - ln1.X);

